pkg/framework/assets/apiserver_ca.crt

```diff
@@ -0,0 +1,23 @@
+-----BEGIN CERTIFICATE-----
+MIIDwzCCAqugAwIBAgIUKlswr/x5/FFyyL7mL7kCPi27dSMwDQYJKoZIhvcNAQEL
+BQAwXDELMAkGA1UEBhMCeHgxCjAIBgNVBAgMAXgxCjAIBgNVBAcMAXgxCjAIBgNV
+BAoMAXgxCjAIBgNVBAsMAXgxCzAJBgNVBAMMAmNhMRAwDgYJKoZIhvcNAQkBFgF4
+MB4XDTE3MTIxMjE5MzAwMFoXDTIyMTIxMTE5MzAwMFowHTEbMBkGA1UEAxMSa3Vi
+ZXJuZXRlcy5kZWZhdWx0MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA
+0Y4WVfNk1cjwiiPQmK6ro0UDnf3qAEZc55Xqa+txVAstpA9pJWpAKAxQGP8Nbl1B
+GAqeqYOib7/2Lr5Ui0LNi9rvN4lNNNuZmeHfIqpDxgv/a6zrMeOkixRTP9yWJxkD
+QX+ZVKJ8Dwa7cpZFhnzGMchs24q4GAKNEl1dzBO5QKWSsUjfBW9i+bXCaAO3WGi/
+9IPFt2ol6vFML7pSHl/cWkK1qqytQxuJPybeeDDc2qQBOKO1FSmQTaFuAvjO2WtB
+zrnu0t7tgsVbE7dT/rgvmZYsf0PN2Ly6u8K0MCZQPBAAAv6/BKQTJt7qyXL2fV5V
+g6hJ4XT9rW5HYj2PehJ0lwIDAQABo4G7MIG4MA4GA1UdDwEB/wQEAwIFoDATBgNV
+HSUEDDAKBggrBgEFBQcDATAMBgNVHRMBAf8EAjAAMB0GA1UdDgQWBBRXlgiiXDCv
+BqR3cnlFLOR3luTRqjAfBgNVHSMEGDAWgBSkSamcYNBiEpi2xNQ93XiuUC+OfDBD
+BgNVHREEPDA6ghZrdWJlcm5ldGVzLmRlZmF1bHQuc3Zjgglsb2NhbGhvc3SCCWxv
+Y2FsaG9zdIcEfwAAAYcECgAAATANBgkqhkiG9w0BAQsFAAOCAQEAm215zaH5dwRr
+AnSQQ1x0QnTiMbTgmZucYpHWgZ8QrNsC5ioypHtwbSwEbjzSxmddFu2ZEXNcDTtj
+aWU9M8j0JMSOSykxltOTIcnG5aCuH2IXsG5l0F/6Q5iXUUQu+nKw89xY396DxSGe
+jvvrXgKu3iTadv5/N0QuNmj8+bPk9B9ELYNFiYzk64kbmr0/elXaMaPA6g/1XvqB
+WjWmyvFisfZpuBuNGYLAi21qKIKUEItH71w120AbuR14x8YkK/Afp1+mh67gDgdL
+Rcggy/IJOIuqJLvynmfb17iED+7HVw/4ujkg5LbLSC1iQLO8HjlNUu/dvSIg6oHQ
+0UUz9d+Yzg==
+-----END CERTIFICATE-----
```

Comment:
Looks like this is test data. It will better to have them in directory testData to be more explicit ?